File: python/explore.py
from twilio.rest import Client

client = Client('xxxxxxxxxx', 'xxxxxxxxxxx')

# list messages
for msg in client.messages.list():
    print(msg.body)

# delete messages
# for msg in client.messages.list():
# 	print(f"Deleting {msg.body}")
# 	msg.delete()


# create messages
# msg = client.messages.create(
# 		to='xxxxxxxx',
# 		from_='xxxxxxxx',
# 		body='Hello from python!'
# 	)

# print(f'created a message: {msg.sid}')

File: js/explore.js
const Twilio = require('twilio');

const client = new Twilio('xxxxxxxxxxxxxxx','xxxxxxxxxxxxx')

client.messages
.list()
.then(messages => 
	console.log(`The most recent message is ${messages[0].body}`)
	).catch(err => console.error(err));

console.log('gathering messages')

